packages/server/src/service/trpc.ts

```diff
@@ -36,6 +36,11 @@ export class VbmRpc implements ImplRpc<IVbmRpc, Request> {
       googleId: orgObj?.googleId,
     })
   }
+  public fetchFeatureFlags = async () => {
+    return data({
+      REACT_APP_EMAIL_FAX_OFFICIALS: Number(processEnvOrThrow('REACT_APP_EMAIL_FAX_OFFICIALS'))
```

Comment:
So this would be `!!processEnvOrThrow('REACT_APP_EMAIL_FAX_OFFICIALS')`

Reply:
done